mlos_core/mlos_core/tests/optimizers/optimizer_test.py

```diff
@@ -48,7 +48,7 @@ def test_create_optimizer_and_suggest(configuration_space: CS.ConfigurationSpace
 
     assert optimizer.parameter_space is not None
 
-    suggestion = optimizer.suggest()
+    suggestion, _ = optimizer.suggest()
```

Comment:
here and in other places:
```suggestion
    suggestion, _metadata = optimizer.suggest()
```
it's better to spell out things